<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="api.tests.test_view_auth.IsRedHatInternalUserTestCase-20250630054001" tests="2" file="api/tests/test_view_auth.py" time="0.001" timestamp="2025-06-30T05:40:51" failures="0" errors="0" skipped="0">
	<testcase classname="api.tests.test_view_auth.IsRedHatInternalUserTestCase" name="test_has_is_internal_permission" time="0.001" timestamp="2025-06-30T05:40:51" file="advisor/api/tests/test_view_auth.py" line="378"/>
	<testcase classname="api.tests.test_view_auth.IsRedHatInternalUserTestCase" name="test_has_is_internal_permission_false" time="0.001" timestamp="2025-06-30T05:40:51" file="advisor/api/tests/test_view_auth.py" line="400"/>
</testsuite>
